app/tournamentBusiness/tournamentAlerts.js

```diff
@@ -0,0 +1,40 @@
+import bootstrap from "bootstrap";
+import $ from "jquery";
+
+export function showNotEnoughPlayers() {
+    showCustomAlert(
+        "Not enough players",
+        "The tournament cannot be arrange with less than 2 players."
+    );
+}
+
+export function showValidationError(validationResult) {
+    showCustomAlert(
+        "Not a valid player",
+        validationResult.error
+    )
+}
+
+export function showMatchNotFinished() {
+    showCustomAlert(
+        "The match has not even started yet!",
+        "Please be patient."
+    );
+}
+
+export function showWinner(player) {
+    showCustomAlert(
+        "Holy cats!",
+        "player.name won the tournament!"
+    )
+}
+
+export default function showCustomAlert(title, msg) {
```

Comment:
That function is the only one which can be used whenever a user wanted, independently of the other specific message error. But it really does not mean anything, I just have not used any other default export in the whole project.